src/cloudflare/internal/images-api.ts

```diff
@@ -68,19 +85,38 @@ class ImageTransformerImpl implements ImageTransformer {
     return this;
   }
 
-  public async output(
-    options: ImageOutputOptions
-  ): Promise<ImageTransformationResult> {
-    if (this.consumed) {
-      throw new ImagesErrorImpl(
-        'IMAGES_TRANSFORM_ERROR 9525: ImageTransformer consumed; you may only call .output() once',
-        9525
+  public draw(
+    image: ReadableStream<Uint8Array> | ImageTransformer,
+    options?: ImageDrawOptions
```

Comment:
With this, you can get rid of the `|| {}` checks

```suggestion
    options: ImageDrawOptions = {}
```